docs/docs/cmd/pp/dataverse/dataverse-table-row-remove.md

```diff
@@ -0,0 +1,52 @@
+# pp dataverse table row remove
+
+Removes a row from a dataverse table in a given environment
+
+## Usage
+
+```sh
+pp dataverse table row remove [options]
+```
+
+## Options
+
+`-e, --environment <environment>`
+: The name of the environment to remove a row from a table from.
+
+`-i, --id <id>`
+: The id of the row to remove.
+
+`-t, --tablename <tablename>`
```

Comment:
actually it should be 

```suggestion
`-t, --tableName <tableName>`
```

so it will be aligned with `pp dataverse table row list` command